File: 2085/python3/main.py
from collections import Counter

class Solution:
    def countWords(self, words1: List[str], words2: List[str]) -> int:
        p1, p2 = Counter(words1), Counter(words2)

        result = 0
        for word in p1:
            if p1[word] == 1 and p2[word] == 1:
                result += 1
        return result

File: 2085/go/main.go
func countWords(words1 []string, words2 []string) int {
	m1, m2 := make(map[string]int, 0), make(map[string]int, 0)
	for _, word := range words1 {
		m1[word]++
	}
	for _, word := range words2 {
		m2[word]++
	}
	result := 0
	for word, count := range m1 {
		if count == 1 && m2[word] == 1 {
			result++
		}
	}
	return result
}
